components/compact-log-backup/src/compaction/meta.rs

```diff
@@ -283,15 +286,18 @@ impl CompactionRunInfoBuilder {
     async fn find_expiring_files(
         &self,
         s: Arc<dyn ExternalStorage>,
+        until_ts: u64,
     ) -> Result<Vec<ExpiringFilesOfMeta>> {
         let ext = LoadFromExt::default();
         let mut storage = StreamMetaStorage::load_from_ext(&s, ext).await?;
 
         let mut result = vec![];
         while let Some(item) = storage.try_next().await? {
-            let exp = self.expiring(&item);
-            if !exp.is_empty() {
-                result.push(exp);
+            if item.min_ts <= until_ts {
```

Comment:
what's the meaning of `min_ts` here? should we check max_ts too ?

Reply:
if `min_ts` of the metadata file is larger than the compaction task `until_ts`, no kv will be chosen to compact. Therefore, there is no need to record it into migration.